t/data/24-worker-settings/workers.ini

```diff
@@ -13,6 +13,8 @@ WORKER_CLASS = qemu_i386,qemu_x86_64
 [2]
 WORKER_CLASS = qemu_aarch64  
 FOO = bar
+RETRY_DELAY = 10
+RETRY_DELAY_IF_WEBUI_BUSY = 120
```

Comment:
could you please also mention the config variables that are possible in https://github.com/os-autoinst/openQA/blob/master/etc/openqa/workers.ini ? Potentially add more missing?

Reply:
I guess there are many more missing. Should I repeat that variables in all scopes which they can possibly occur. Maybe I just put them in global space and mention that they can occur in instance and web UI scope as well.